src/utils/handle-template-flags.js

```diff
@@ -0,0 +1,11 @@
+const getBaseOptions = require('./get-base-options');
```

Comment:
do we really need this fn as a util? it doesn't even have tests since they are covered by the fn that consumes it (aka generate-from-template). I would just move it into `generate-from-template`

Reply:
its because of complexity that i put this into a util function